stacks_queues/stack.py

```diff
@@ -21,18 +23,23 @@ def pop(self):
             The Stack is empty.
             returns None
         """
-        pass
+        if not self.store:
```

Comment:
`store` is initialized to a LinkedList instance at construction, and should never be `None`. But we could check whether it's empty
```suggestion
        if self.store.empty():
```